plugins/filter/jc.py

```diff
@@ -26,6 +26,7 @@
       description:
         - The correct parser for the input data.
         - For example C(ifconfig).
+        - Note: use underscores instead of dashes (if any) in the parser module name
```

Comment:
```suggestion
        - "Note: use underscores instead of dashes (if any) in the parser module name."
```
Otherwise this defines a YAML dictionary.